demo/utilities/colorMode.ts

```diff
@@ -0,0 +1,24 @@
+import { useLocalStorage } from '@prefecthq/vue-compositions'
```

Comment:
This file would make a nice composition. Don't bother but would be cool at some point.

Reply:
Agreed, I think I've created this or something like it in a few places now